src/components/StackedNotification/StackedNotification.js

```diff
@@ -45,7 +46,7 @@ StackedNotification.propTypes = {
   title: PropTypes.string.isRequired,
   subtitle: PropTypes.node.isRequired,
   role: PropTypes.string,
-  caption: PropTypes.node,
+  caption: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```

Comment:
A `node` type includes a `string` so no need to change this.

```suggestion
caption: PropTypes.node,
```

Reply:
Wasn't aware of that @jendowns, thanks! I just changed it back.